### DISCUSSION TITLE
Kernel folder

### DISCUSSION THREAD

**Discussion by boazkalk** [daducci/COMMIT#135] [General]
Hi all,

The commit pipeline creates a "kernels" folder in the main folder the script is located in. Is there a way to change the location this kernel folder is placed? I want to run Commit on an HPC in parallel for a lot of patients so I want to put the kernels folder in a single patient's output folder.

Thanks in advance!

**Comment by daducci** [comment]
Dear @boazkalk ,

If you have all your subjects stored in one single folder, then you can use the `study_path` and `subject` parameters in the call `mit = commit.Evaluation(study_path='<PATH>', subject='<PATH>')` when you process each single subject, and then the `kernel` folder will be created only in the `study_path` folder. Does this solve your problem?
